docker-assets/seed_database

```diff
@@ -9,6 +9,25 @@ function write_encryption_key() {
 KEY
 }
 
+function check_svc_status() {
+  local SVC_NAME=$1 SVC_PORT=$2
+
+  [[ $# -lt 2 ]] && echo "Error something seems wrong, we need at least two parameters to check service status" && exit 1
+
+  echo "== Checking ${SVC_NAME}:$SVC_PORT status =="
+
+  while true; do
+    ncat ${SVC_NAME} ${SVC_PORT} < /dev/null && break
+    sleep 5
+  done
+  echo "${SVC_NAME}:${SVC_PORT} - accepting connections"
+}
+
 write_encryption_key
 
+# Wait for postgres to be ready
+check_svc_status $DATABASE_HOST $DATABASE_PORT
+
 bundle exec rake db:migrate db:seed
+
+exec "$@"
```

Comment:
So I'm pretty sure I get why this is needed - its basically so we can run `entrypoint seed_database run_rails_server` rather than `entrypoint seed_database && entrypoint run_rails_server`. I just want to validate that it doesn't exit with exit code 1 without any args which would indicate failure in our initContainer. I'll test it locally and validate.

Reply:
Yep! Without this, we'd exit locally when the seed would run.